modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/configuration/DLSizeLimitConfigurationProviderImpl.java

```diff
@@ -45,6 +47,52 @@ public Map<String, Long> getCompanyMimeTypeSizeLimit(long companyId) {
 			companyId);
 	}
 
+	@Override
```

Comment:
These two new methods should not go in the configuration provider. These are not providing configuration values, but doing validations. You should move these to a new helper class private to the module, unless they're used only once (in that case, inline in the class that is using the method).

Also, about the `getCopyToFailInfo` name, I think this should be changed to something less ambiguous, like `getCopyToValidationMessage`.